cluster-autoscaler/cloudprovider/clusterapi/README.md

```diff
@@ -169,8 +169,7 @@ cluster-autoscaler --cloud-provider=clusterapi \
 
 To enable the automatic scaling of components in your cluster-api managed
 cloud there are a few annotations you need to provide. These annotations
-must be applied to either [MachineSet](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/machine-set.html)
-or [MachineDeployment](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/machine-deployment.html)
+must be applied to either [MachineSet](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/machine-set.html), [MachineDeployment](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/machine-deployment.html), or [MachineDeployment](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/machine-deployment.html)
```

Comment:
Looks like we have `MachineDeployment` listed twice here, is one supposed to be `MachinePool`?

Reply:
Thanks for reading closely–I must have failed to save after a copy-paste. Fixed.

Reply:
Sure thing, super excited to get this feature, thanks so much for tackling it! 🎉